frontend/config/webpack.config.dev.js

```diff
@@ -182,7 +182,7 @@ module.exports = {
                   plugins: () => [
                     require('postcss-flexbugs-fixes'),
                     autoprefixer({
-                      browsers: [
+                      overrideBrowserslist: [
```

Comment:
I'm not familiar with this, but they seem to suggest [here](https://github.com/postcss/autoprefixer/blob/master/README.md#options) that they recommend not using this option.

Reply:
Alright, I've moved this to package.json and changed it to use "defaults" instead